lib/fetchStreamReadable/parser/chunksParser.js

```diff
@@ -86,6 +88,9 @@ function parse({ status, callbacks , isFirst = false, separator }) {
       }
       catch (error) {
         if (error)  {
+          logger(`browser-sdk :: chunksParser :: parse :: id = ${status.id} :: errorPared = ${status.state}:: errorMsg = ${error.msg} ::  bufferString = ${status.bufferString}`);
```

Comment:
In my mind this line would must be after joinMsgsIfObjectHasStrings(error);
Also there is a typo in 'errorPared'